src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFiles.java

```diff
@@ -186,11 +186,11 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
             return;
         }
 
-
-        // create VET output
-        if (!variant.isReferenceBlock()) {
+        // write to VET if NOT reference block and NOT a no call (set GQ to 0 for variant)
+        if (!variant.isReferenceBlock() && !variant.getGenotype(0).isNoCall()) {
```

Comment:
why `getGenotype(0)`? what is 0?

Reply:
0 is the index/position of what getGenotype returns for `variant` and in this case it gives back the GT for `variant` which then is used to check if its a no-call. aka is ./. = no-call for those that are no calls

Reply:
if a variant is a no call, it should have only one genotype where isNoCall is true.

Reply:
so is that to say, it's safe to only look at the first item in the list of GTs? a sample could have more than one GT?

Reply:
the code is a little hard to track down, but I do think that for isNoCall to be set to true, this must hold
`    public static boolean wouldBeNoCallAllele(final byte[] bases) {
        return bases.length == 1 && bases[0] == htsjdk.variant.vcf.VCFConstants.NO_CALL_ALLELE;
    }
`

Reply:
`variant.getGenotype()` returns a list of lists, one list for each sample. if we are certain that there will always be exactly one sample per gvcf being processed here, then this is safe. I just don't know if this is always going to be true.

Reply:
we added a constant and some comments to make this assumption super clear

Reply:
This tools runs on a single-sample gVCF, I think it's totally reasonable to assume there is a single genotype (since there is a single sample and a sample can only have one genotype)